driver-mapping/src/test/java/com/datastax/driver/mapping/MapperInvalidAnnotationsTest.java

```diff
@@ -261,27 +265,4 @@ public void setNotReadable(int i) {
 
     }
 
-    @Test(groups = "unit", expectedExceptions = IllegalArgumentException.class,
```

Comment:
It looks like in the case of unreadable it throws an NPE (on a Preconditions check) and in the case of unwritable it logs:

> [main] WARN  com.datastax.driver.mapping.config.DefaultPropertyAccessStrategy  - Property 'notWritable' is not writable and will not be mapped

but doesn't throw an exception and returns an `EntityMapper` without any columns.